src/Extensions/WorkflowApplicable.php

```diff
@@ -319,6 +327,19 @@ public function LinkToPendingItems()
      */
     public function onAfterWrite()
     {
+        // Record changes to the workflow on this record
+        $changed = $this->owner->getChangedFields(['WorkflowDefinitionID']);
+        if (isset($changed['WorkflowDefinitionID']) &&
+            (
+                empty($changed['WorkflowDefinitionID']['before'])
+                || empty($changed['WorkflowDefinitionID']['before'])
```

Comment:
Are these the same thing?

Reply:
It appears to be, yep. Perhaps the first was supposed to be `isset`.

Reply:
`isset` would be covered by `empty` anyway, maybe we just need the one